src/CloudFoundry.Test.Integration/EnvVariables.cs

```diff
@@ -0,0 +1,129 @@
+﻿using CloudFoundry.CloudController.V2.Client;
+using CloudFoundry.CloudController.V2.Client.Data;
+using CloudFoundry.UAA;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CloudFoundry.Test.Integration
```

Comment:
These are not integration tests. They don't test the source code in the repo, but whatever may be deployed remotely.